### ISSUE TITLE
Does VoxelStats always need a "distributed computing server"?

### ISSUE THREAD

**Issue by Tkhr02** [sulantha2006/VoxelStats#9]
Hello!!

Do I need to have a MATLAB "distributed computing server" in order to use VoxelStats for processing? 
By the way, I have the "parallel computing toolbox" in my MATLAB.

In my lab, the "distributed computing server" is very expensive and I cannot afford it.

If we don't need to use a "distributed computing server", we will be very grateful.

**Comment by SulanthaM** [comment]
No. You can run VoxelStats without a distributed license. It will run in a single machine using all the cores it has (or I believe however you have set up the local parallel profile)

**Comment by Tkhr02** [comment]
Thanks for the reply.

I have one more question.
Other than the "parallel computing toolbox", is there any other MATLAB toolbox that I need to use VoxelStats?

**Comment by SulanthaM** [comment]
The main one is stats toolbox, which I believe is present by default. 
Best guess is, try running something and it will complain if something is missing :-)

**Comment by Tkhr02** [comment]
You're certainly right. I will deal with the error when it comes up.

Thank you for answering my question in such a polite way.

**Comment by SulanthaM** [comment]
You are welcome and let me know if you run into more trouble.

**Comment by Tkhr02** [comment]
I’m counting on you.
